packages/react-components/src/components/DomainPermissions.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Stack, Text, Link, Icon } from '@fluentui/react';
+/* @conditional-compile-remove(call-readiness) */
+import { useLocale } from '../localization';
+/* @conditional-compile-remove(call-readiness) */
+import { _formatString, _pxToRem } from '@internal/acs-ui-common';
+import {
+  iconBackDropStyles,
+  iconContainerStyles,
+  iconPrimaryStyles,
+  linkTextStyles,
+  primaryTextStyles,
+  secondaryTextStyles,
+  textContainerStyles
+} from './styles/DomainPermissions.styles';
+
+/**
+ * @internal
+ * Props for DomainPermissions component.
+ */
+export interface DomainPermissionsProps {
+  appName: string;
+  onGetTroubleShooting: () => void;
+  strings: DomainPermissionsStrings;
+}
+
+/**
+ * @beta
+ * Strings for DomainPermissions component
+ */
+export interface DomainPermissionsStrings {
+  mainText: string;
+  secondaryText: string;
+  linkText: string;
+}
+
+const _DomainPermissionsContainer = (props: DomainPermissionsProps): JSX.Element => {
+  const { appName, onGetTroubleShooting, strings } = props;
+
+  const appNameTrampoline = (): string => {
+    /* @conditional-compile-remove(call-readiness) */
+    return _formatString(strings.mainText, { appName: appName });
+    // For Conditonal compilation will be undone when stable.
+    return appName;
+  };
+  const containerWitdthTrampoline = (): string => {
+    /* @conditional-compile-remove(call-readiness) */
+    return `${_pxToRem(406)}`;
```

Comment:
why is this line conditional compiled?

Reply:
the imports complain without trampolining this. I can always just do the math to find this min in rem?